clang/test/SemaSYCL/sycl-attr-warn-non-kernel.cpp

```diff
@@ -1,4 +1,4 @@
-// RUN: %clang_cc1 -fsycl-is-device -internal-isystem %S/Inputs -fsyntax-only -verify -pedantic %s
+// RUN: %clang_cc1 -fsycl-is-device -Wno-error=deprecated-declarations -internal-isystem %S/Inputs -fsyntax-only -verify -pedantic %s
```

Comment:
I feel like introducing an extra `expected-warning` would be a clearer approach here

Reply:
Done